lib-src/libnyquist/nyquist/nyqsrc/sound.c

```diff
@@ -1591,7 +1591,7 @@ double step_to_hz(double steps)
 }
```

Comment:
There have been a number of changes to Nyquist since it was last merged (and since this pull request and since I really looked at this stuff).